### PR TITLE
review data index page

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/framework/data/index.md [hunk 1]
```diff
@@ -1,43 +1,37 @@
...
   - ".NET Framework, data access"
   - "data access [.NET Framework], about .NET Framework data access"
   - "data [.NET Framework], accessing"
 ms.assetid: 8c37635d-e2c1-4b64-a258-61d9e87405e6
 ---
-# Data and Modeling in the .NET Framework
-This section provides information on how to access data using ADO.NET, Language Integrated Query (LINQ), WCF Data Services, and XML.  
+# Data and modeling in the .NET Framework
+
+This section provides information on how to access data in the .NET Framework.
```
Comment (rpetrusha) [inline docs/framework/data/index.md]: Most of this applies to .NET Core as well, so .NET Framework --> .NET?

File: docs/framework/data/index.md [hunk 2]
```diff
@@ -1,43 +1,37 @@
...
+## In this section
+
+ [WCF Data Services 4.5](./wcf/index.md)  
+ Provides information about how to use WCF Data Services to deploy data services on the Web or an intranet.  
+
  [ADO.NET](./adonet/index.md)  
  Describes the ADO.NET architecture and how to use the ADO.NET classes to manage application data and interact with data sources, including Microsoft SQL Server, OLE DB data sources, and XML.  
   
+ [Transaction Processing](./transactions/index.md)  
+ Discusses the .NET Framework support for transactions.
```
Comment (rpetrusha) [inline docs/framework/data/index.md]: .NET, or .NET Framework?

**Review by rpetrusha — APPROVED** [review]
Left a couple of questions/suggestions, @mairaw.

**Comment by mairaw** [comment]
Good point @rpetrusha. I've changed to .NET.

**Pull request merged**